main/cogs/Dnd.py

```diff
@@ -182,6 +182,34 @@ def check(message):
 
         await ctx.send(file=file)
 
+    # ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    #                                   Scroll gen
+    @commands.command(name="gscroll")
+    async def gscroll(self, ctx, *args):
```

Comment:
async def gscroll(self, ctx, *, args):

Reply:
This will remove the need of having to interate through them and add them to the spell string.